modules/assignment/src/main/java/com/intuit/wasabi/assignment/cache/impl/NoopAssignmentsMetadataCacheImpl.java

```diff
@@ -0,0 +1,193 @@
+import com.intuit.wasabi.experimentobjects.Page;
+import com.intuit.wasabi.experimentobjects.PageExperiment;
+import com.intuit.wasabi.experimentobjects.PrioritizedExperimentList;
+import com.intuit.wasabi.repository.CassandraRepository;
+import com.intuit.wasabi.repository.ExperimentRepository;
+import com.intuit.wasabi.repository.MutexRepository;
+import com.intuit.wasabi.repository.PagesRepository;
+import com.intuit.wasabi.repository.PrioritiesRepository;
+import net.sf.ehcache.Cache;
+import net.sf.ehcache.CacheManager;
+import net.sf.ehcache.Element;
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import static com.intuit.wasabi.assignment.cache.AssignmentsMetadataCache.CACHE_NAME.APP_NAME_N_PAGE_TO_EXPERIMENTS_CACHE;
+import static com.intuit.wasabi.assignment.cache.AssignmentsMetadataCache.CACHE_NAME.APP_NAME_TO_EXPERIMENTS_CACHE;
+import static com.intuit.wasabi.assignment.cache.AssignmentsMetadataCache.CACHE_NAME.APP_NAME_TO_PRIORITIZED_EXPERIMENTS_CACHE;
+import static com.intuit.wasabi.assignment.cache.AssignmentsMetadataCache.CACHE_NAME.EXPERIMENT_ID_TO_BUCKET_CACHE;
+import static com.intuit.wasabi.assignment.cache.AssignmentsMetadataCache.CACHE_NAME.EXPERIMENT_ID_TO_EXCLUSION_CACHE;
+import static com.intuit.wasabi.assignment.cache.AssignmentsMetadataCache.CACHE_NAME.EXPERIMENT_ID_TO_EXPERIMENT_CACHE;
+import static java.util.Objects.isNull;
+
+/**
+ *  Local cache created and used during user assignment flow.
+ *
+ */
+
+public class NoopAssignmentsMetadataCacheImpl implements AssignmentsMetadataCache {
+    private final Logger LOGGER = LoggerFactory.getLogger(NoopAssignmentsMetadataCacheImpl.class);
+
+    @Inject
+    public NoopAssignmentsMetadataCacheImpl() {
+        LOGGER.info("Noop Assignments metadata cache has been created successfully...");
+    }
+
+    /**
+     * This method is used to clear cache.
+     */
+    @Override
+    public boolean clear() {
+        // NOOP
+       return Boolean.FALSE;
+    }
+
+    /**
+     * This method refresh the existing cache (keys) with the updated data from Database.
+     *
+     * This method doesn't add new keys into the cache.
+     *
+     * @return TRUE if cache is successfully refreshed else FALSE.
+     */
+    @Override
+    public boolean refresh() {
+        // NOOP
+        return Boolean.FALSE;
+    }
+
+    /**
+     * @param appName
+     *
+     * @return List of experiments created in the given application.
+     */
+    @Override
+    public List<Experiment> getExperimentsByAppName(Application.Name appName) {
+        // NOOP
```

Comment:
This is just a NOOP implementation so returning NULL and in real implementation this method will never return NULL as core assignment logic expects NON-NULL value only...